### ISSUE TITLE
Combobox: `role="combobox"` attribute is on the wrong element causing Accessibility Checker error

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#7331]
background https://github.com/IBMa/equal-access/issues/282#issue-747729478

Our Combobox has a wrapper div with the attribute `role="combobox"`. This attribute should be on the input itself. This is causing the below AC violation. 
![image](https://user-images.githubusercontent.com/40970507/99859640-39b0c500-2b56-11eb-9624-4b85bb6edb68.png)

Sort of a weird error but essentially since AC expects -- as per WCAG recommendation -- for the combobox role to be on the input and it expects that input to receive focus first in the DOM AND instead something else gets focus -- we get this error. 

Moving that attribute to the input itself should fix this issue.

**Comment by JensenAlexander** [comment]
Is there a workaround for this?

**Comment by tw15egan** [comment]
Seem to also be related to issues seen in `Filterable Multiselect` when the menu is open and items are selected (#7296 and #6570)

- Initial DOM focus on the combobox is not set on the text input when the combobox receives focus
- The element using 'aria-activedescendant' set to "carbon-multiselect-example-item-1" does not have its 'tabindex' attribute value set to 0 or -1

**Comment by dakahn** [comment]
closing as a dupe of https://github.com/carbon-design-system/carbon/issues/6938
